Controllers/controller.py

```diff
@@ -45,6 +45,9 @@ def __init__(self, window):
 
         self._window.table_panel.add_col_button.clicked.connect(self.add_col_to_encoding_table)
         self._window.table_panel.add_row_button.clicked.connect(self.add_row_to_encoding_table)
+        self._window.media_panel.media_control_panel.playBtn.clicked.connect(self.play_video)
+
```

Comment:
Lines 49 and 51 should be removed. Functions should be separated by only 1 blank line.